src/isIsogram.test.js

```diff
@@ -8,12 +8,18 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
-
+    expect(typeof isIsogram('mate')).toEqual('boolean');
   });
 
   it(`should be case sensitive`, () => {
+    expect(isIsogram('MATE')).toBeTruthy();
+  });
```

Comment:
In readme file writen: "function not case sensitive (`M` and `m` are considered the same letter)."
So maybe it is better to change a little:
 it(`should not be case sensitive`, () => {
    expect(isIsogram('MmNn')).toBeFalsy();
  });

Reply:
That's right, you don't check case sensitivity here. Try to follow Dmytro's recommendation